resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesDriverBuilder.scala

```diff
@@ -71,8 +75,9 @@ private[spark] class KubernetesDriverBuilder(
         case JavaMainAppResource(_) =>
           provideJavaStep(kubernetesConf)
         case PythonMainAppResource(_) =>
-          providePythonStep(kubernetesConf)}
-      .getOrElse(provideJavaStep(kubernetesConf))
+          providePythonStep(kubernetesConf)
+        case RMainAppResource(_) =>
+          provideRStep(kubernetesConf)}.getOrElse(provideJavaStep(kubernetesConf))
```

Comment:
I'd move `}.getOrElse(provideJavaStep(kubernetesConf))` to the next line for clarity?

Reply:
done